index.html

```diff
@@ -100,6 +100,48 @@ <h1 class="display-3 text-center" id="directorHeading">D I R E C T O R</h1>
       </div>
     </div>
 
+    <div class="container">
+      <div class="row mt-5">
+        <div class="col-12 text-center">
+          <p>As a trusted partner, Jetpharma offers</p>
+          <h1 class="display-3" id="servicesHeading">S E R V I C E S</h1>
+        </div>
+      </div>
+      <div class="row" id="servicesTextRow">
+        <div class="col-lg-4 px-5">
+          <img src="assets/regulatory-affairs-icon.svg" alt="Regulatory Affairs Icon" class="servicesIconImages rounded mx-auto d-block">
```

Comment:
there isn't a must follow convention for this but we usually prefer to use hyphens instead of camelCase for classes name. e.g `services-icon-images`

for id then camelCase is fine